### PR TITLE
MTP-1862-utlisting-av-forsoksdyrsammendrag

### PR DESCRIPTION
**Pull request by amundlindstol**

### CODE REVIEW

**Comment by petterkoding** [comment]
Tested in storybook. Looks good.

File: mt-kit/core/svelte/src/lib/svelte/components/Table.svelte [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  let className = ''
+  export { className as class }
+  export let style = ''
+  export let caption = ''
+  export let headers: Array<{}> = [{}]
+  export let rows: Array<{}> = [{}]
+</script>
+
+{#if caption}
+  <caption class="m-t-m">
```
Comment (lha047) [inline mt-kit/core/svelte/src/lib/svelte/components/Table.svelte:11]: I think we should pass spacing to component. Might not always need it

File: mt-kit/core/svelte/src/lib/svelte/components/Table.svelte [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    {caption}
+  </caption>
+{/if}
+<table class="mt-table responsive-table m-t-xxs {className}" id="table-{caption}" {style}>
```
Comment (lha047) [inline mt-kit/core/svelte/src/lib/svelte/components/Table.svelte:15]: `m-t-xxs` same as above

**Review by lha047 — CHANGES_REQUESTED** [review]
Looks good :) 

Just a few comments

**Review by lha047 — APPROVED** [review]
Nice :D

**Pull request merged**